src/bundle/Resources/views/themes/admin/ui/component/tag_view_select/tag_view_select_selected_item_tag.html.twig

```diff
@@ -4,6 +4,7 @@
 <div class="ibexa-tag-view-select__selected-item-tag" data-id="{{ id }}" data-name="{{ name }}">
```

Comment:
If I understand correctly what you want to achieve, you should add here a class `ibexa-tag-view-select__selected-item-tag--not-deletable` and style middle-ellipsis by this class.
Middle-ellipsis is a component that doesn't know anything about the deletable item, so the logic for it should be here.